debugging/helper.js

```diff
@@ -490,7 +492,16 @@ const extractDatabases = function (info, dump) {
         };
 
         database.collections.push(coll);
-        info.collections[full] = coll;
+
+        if (_.has(info.collections, full)) {
+          info.obsoleteCollections.push(full);
```

Comment:
I must admit I do not know the details of how this enumerated here, to me this looks a bit of (but may still be okay)
Assume `full` has a very small ID.
Then we push it in into obsolete collections multiple times.
While we skip `coll` entirely.
Is there some order guaranteed?
Is the idea here that we have all collections ordered either in info.collections[] or obsoleteCollection or is it okay to miss some?

If we need all entries to be added into one of the lists, I think the push to obsoleteCollections should be done within the IF a couple of lines below, to ensure we put the lower id into collections and the higher into obsolete always.

Reply:
Let me check if we need to check for multiply pushes

Reply:
Is OK to record all